app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,31 @@
+module Api
+  module V1
+    class BookSuggestionsController < DeviseTokenAuthController
+      rescue_from ActiveRecord::RecordInvalid, with: :on_invalid_record
+
+      def create
+        suggestion = BookSuggestion.create! book_suggestion_params
+        respond_to do |format|
+          format.json { render status: :created, json: suggestion }
+        end
+      end
+
+      def book_suggestion_params
+        result = params.fetch(:book_suggestion, {}).permit(
```

Comment:
I would use require since you always need the book_suggestion and not getting book_suggestion would be an error.